src/Symfony/Bundle/TwigBundle/TemplateIterator.php

```diff
@@ -18,6 +18,8 @@
  * Iterator for all templates in bundles and in the application Resources directory.
  *
  * @author Fabien Potencier <fabien@symfony.com>
+ *
+ * @internal
```

Comment:
Should be `@internal since Symfony 4.4` + CHANGELOG/UPGRADE files should be updated too, right?

Reply:
done